src/Propulsion.EventStore/Propulsion.EventStore.fsproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard2.0|AnyCPU'">
-    <DefineConstants>TRACE</DefineConstants>
+    <DefineConstants>TRACE;EVENTSTORE</DefineConstants>
```

Comment:
make EVENTSTORE the default - i.e. remove all custom stuff from the ES side and make SqlStreamStore the special case